docs/cql/ddl.rst

```diff
@@ -769,12 +773,8 @@ available:
                                            for a read. Allowed values are powers of two between 1 and 128.
 ========================= =============== =============================================================================
 
-.. ``crc_check_chance``      1.0             When compression is enabled, each compressed block includes a checksum of
-..                                           that block for the purpose of detecting disk bitrot and avoiding the
-..                                           propagation of corruption to other replicas. This option defines the
-..                                           probability with which those checksums are checked during read. By default
-..                                           they are always checked. Set to 0 to disable checksum checking and to 0.5 for
-..                                           instance to check them every other read   |
+.. crc_check_chance was promoted to a top-level table option since Cassandra 3.0,
+   so it is documented along with other table options.
```

Comment:
Sorry, my previous comment may have been misleading. Cassandra did promote this to top-level, but we *didn't* yet do this.